Nanotrasen Advantage / Sharplite

rye-rice opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

The long awaited Sharplite Rework and a few other less memoriable weapons

After this the gun rework should almost be done, minus attachments and a few more i wanted to do

<img width="142" height="143" alt="Screenshot_20250911_170532" src="https://github.com/user-attachments/assets/436e5107-9581-4a07-af7a-2720cb1f6f47" />

Nanotrasen ballistics fill the same niche as their predecessors, not much to really say here.

Sharplite lasers however have been completely redone, I'm sure theres a lot of fun to be had there

Credits: 
All weapons were designed by apogee
204, 201, x12, x46, al607, al655, al660 - Drawn by Imaginos16
Sound effects - Taken from TGMC
thrax - for helping this get over the finishing line

The big sharplite rework that was teased people who are not me

This also resprites the the ballistics to not look crappy

Once again, none of these are "replacements" nor "removals" the lineup itself was reworked from scratch, so do not complain about bad removals

Misc:
- Makes some weapon firing SFX mono instead of stereo
- Sharplite lasers no longer pass through windows
- Sharplite laser speed has been *slightly* nerfed
- Buffs the laserproof armor slightly, makes it slightly cheaper, resprites it
- Inteq variations for said weapons, unobtainable at the moment
- Splits Eoehoma and Nanotrasen weapon cells into incompatible sets
- Sharplite bolt is now blue
- The generic laser gun is changed to "E-20". I will revise this weapon later, but for now its bulky but shoots eoehoma lasers

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl: rye-rice, Imaginos16, Apogee-dev, 
add: Nanotrasen Advantage
add: Sharplite Defence
balance: Laserproof armor was given a slight buff in exchange for reflecting lasers being much more rare
del: The Nanotrasen Gauss rifle is no longer in black market
del: A bunch of old R&D laser weapons
del: sabr
fix: Some sound effects are no longer stereo
fix: 

/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

rye-rice committed:
start

rye-rice committed:
more ballistiic progress

rye-rice committed:
99% ballistics done

rye-rice committed:
progress on lasers

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

rye-rice committed:
merge

rye-rice committed:
more lasers

rye-rice committed:
alpha

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

rye-rice committed:
twaks

rye-rice committed:
balance

rye-rice committed:
missing inhands

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

rye-rice committed:
better sprites

rye-rice committed:
some more

rye-rice committed:
string chages

Ratvarr commented on code/modules/projectiles/guns/manufacturer/nanotrasen_sharplite/ballistics.dm:
```diff
@@ -88,3 +207,218 @@ NO_MAG_GUN_HELPER(automatic/pistol/commander)
+
+/obj/item/gun/ballistic/revolver/mateba
+	name = "Advantage Rhino"
+	desc = "A very famous high-powered semi-auto revolver. Designed by Nanotrasen Advantage for use in shooting competitions, it has ended up as the de-facto officer weapon of the Nanotrasen aliance. Chambered in .357."
```
Nanotrasen `alliance`

rye-rice replied:
Is there an actual name for the trio? Ingame i see a mixture of "tri corp" and some other terms, but i don't know if theres an official name for it like "Syndicate/The Coalition"

Ratvarr commented:
PR looks like it kills some more recent gun sprites for ammo

FalloutFalcon commented on code/__DEFINES/guns.dm:
```diff
@@ -61,7 +61,7 @@
 #define MANUFACTURER_NANOTRASEN_OLD "an outdated Nanotrasen logo"
 #define MANUFACTURER_NANOTRASEN "the Nanotrasen logo"
-#define MANUFACTURER_VIGILITAS "the Vigilitas Interstellar logo"
+#define MANUFACTURER_VIGILITAS "the Nanotrasen Advantage logo"
```
I dont believe this fits with the current direction of the lore. But also you need to update the define as well.

FalloutFalcon replied:
I am wrong in this regard acctually.

FalloutFalcon reviewed (changes requested):
You cannot bundle min recoil in with this pr.

Erikafox commented on code/modules/cargo/packs/magazines.dm:
```diff
@@ -354,10 +354,9 @@
-	name = "Weapon Cell Crate"
-	desc = "Contains a weapon cell, compatible with laser guns."
+	name = "Universal Laser Gun Cell Crate"
+	desc = "Contains a weapon cell, compatible with all Eoehoma laser guns along with cells compatable with Eoehomas. NOT COMPATABLE WITH NANOTRASEN LASERS!!"
```
```suggestion
	name = "Eoehoma-style Power Cell"
	desc = "Contains a weapon power cell built to be compatible with Eoehoma weapons, and systems that still use Eoehoma's style of cell."
```

Erikafox commented on code/modules/cargo/packs/magazines.dm:
```diff
@@ -369,10 +368,18 @@
 
+/datum/supply_pack/magazine/nt_guncell
+	name = "Sharplite Proprietary Weapon Cell Crate"
+	desc = "Contains a proprietary weapon cell, compatible with most Nanotrasen models. NOTE: This is NOT compatable with legacy Eoehoma models!"
```
```suggestion
/datum/supply_pack/magazine/nt_guncell
	name = "Sharplite Power Cell"
	desc = "Contains a proprietary weapon cell, compatible with most Sharplite energy weapons."
```

Erikafox commented on code/modules/cargo/packs/magazines.dm:
```diff
@@ -369,10 +368,18 @@
-	name = "Upgraded Weapon Cell Crate"
-	desc = "Contains an upgraded weapon cell, compatible with laser guns. For NT use only."
-	contains = list(/obj/item/stock_parts/cell/gun/upgraded)
+	name = "Sharplite Plus Proprietary Weapon Cell"
+	desc = "Contains an upgraded weapon cell, compatible with most Nanotrasen models. For NT use only."
```
```suggestion
	name = "Sharplite Plus Power Cell"
	desc = "Contains an high capacity weapon cell, compatible with most Sharplite energy weapons."
```

rye-rice replied:
The "For NT use only" suffix is used for all Nanotrasen's factional cargo, so i might add that back afterwards unless you think it would be best to remove all instances of "For NT use only"?

Erikafox replied:
I feel like being in faction cargo is already an implication that it should be kept somewhat in house

Erikafox commented on code/modules/power/cell.dm:
```diff
@@ -424,28 +429,25 @@
-	name = "upgraded weapon power cell"
-	icon_state = "ug-cell"
+	name = "upgraded eoehoma power cell"
+	desc = "A high capacity weapon cell. Intended for higher duty weapons, these might fit in lower grade weapons with some luck."
```
```suggestion
	name = "high-capacity eoehoma power cell"
	desc = "A high capacity weapon cell. Intended for use in heavy weapons and the odd piece of personal gear."
```

Erikafox commented on code/modules/power/cell.dm:
```diff
@@ -478,6 +478,40 @@
+
+/obj/item/stock_parts/cell/gun/sharplite
+	name = "Sharplite proprietary weapon cell"
+	desc = "A proprietary weapon cell used exclusively by Sharplite weaponry. However, due to the very large market share of Nanotrasen, some non-Nanotrasen weapons accept this cell."
```
```suggestion
	name = "Sharplite power cell"
	desc = "A proprietary power cell primarily used by Sharplite weaponry. Nanotrasen's large market share has forced some weapon developers to include adapters for these cells"
```

Erikafox commented on code/modules/power/cell.dm:
```diff
@@ -478,6 +478,40 @@
+
+/obj/item/stock_parts/cell/gun/sharplite/plus
+	name = "Sharplite Plus proprietary weapon cell"
+	desc = "An upgraded, proprietary weapon cell used exclusively by Sharplite weaponry. Better than the stock version; these Plus versions are bought by preimum customers or alongside the more energy hungry weapons in Sharplite's lineup."
```
```suggestion
	name = "Sharplite Plus power cell"
	desc = "An high-capacity weapon cell used exclusively by Sharplite weaponry. They are a great improvement over the stock cell, and are frequently sought after by collectors, soldiers, and operators of heavy lasers alike."
```

Erikafox commented on code/modules/power/cell.dm:
```diff
@@ -478,6 +478,40 @@
+
+/obj/item/stock_parts/cell/gun/sharplite/mini
+	name = "Sharplite Compact weapon cell"
+	desc = "A proprietary weapon cell used exclusively by Sharplite weaponry. This is a compact version with less capacity, intended to fit into laser handguns."
```
```suggestion
/obj/item/stock_parts/cell/gun/sharplite/mini
	name = "Sharplite Compact power cell"
	desc = "A compact weapon cell used exclusively by Sharplite weaponry. It holds less charge and is intended for usage in energy handguns."
```

Erikafox commented on code/modules/projectiles/ammunition/energy/laser.dm:
```diff
@@ -51,6 +54,18 @@
+/obj/item/ammo_casing/energy/lasergun/sharplite/dmr
+	projectile_type = /obj/projectile/beam/laser/sharplite/dmr
+	e_cost = 1000 // 10 per regular cell  20 per upgraded cell
+	select_name = "kill"
```
```suggestion
	select_name = "marksman"
```

Erikafox commented on code/modules/projectiles/ammunition/energy/laser.dm:
```diff
@@ -51,6 +54,18 @@
+	fire_sound = 'sound/weapons/gun/laser/heavy_laser.ogg'
+	delay = 1.3 SECONDS
+	e_cost = 4000 // 5 per upgraded cell
+	select_name = "kill"
```
```suggestion
	select_name = "sniper" 
```
```suggestion
	select_name = "long distance"
```

both are better than kill imo

rye-rice replied:
neither of these are seen in game, it only affects the sprite name, but if there was a dual select i would probably change it to sniper

Erikafox commented on code/modules/projectiles/ammunition/energy/laser.dm:
```diff
@@ -107,15 +123,15 @@
 	pellets = 3
 	variance = 25
 	e_cost = 1000
 	select_name = "kill"
```
```suggestion
	select_name = "EP scatter"
```

may as well standardize it

Erikafox commented on code/modules/projectiles/ammunition/energy/laser.dm:
```diff
@@ -78,9 +93,10 @@
 	projectile_type = /obj/projectile/beam/weak/sharplite
-	e_cost = 799 //12 shots with a normal power cell, 25 with an upgraded
+	e_cost = 500 //20 shots with a normal power cell, 40 with an upgraded
 	select_name = "kill"
```
```suggestion
	select_name = "light"
```

Erikafox commented on code/modules/projectiles/guns/energy/energy_gun.dm:
```diff
@@ -129,72 +39,9 @@
 /obj/item/gun/energy/e_gun/rdgun
 	name = "research director's PDW"
-	desc = "A energy revolver made from the power of science, but more importantly booze. Only has 6 shots."
+	desc = "A custom energy revolver made from the power of science, but more importantly booze. Only has 6 shots. It looks well worn, however that was long, long ago."
```
```suggestion
	desc = "A custom energy revolver made with scientific intent, but more importantly booze.  That was long ago. It's just a curiosity now."
```

Erikafox commented on code/modules/projectiles/guns/energy/laser.dm:
```diff
@@ -1,6 +1,7 @@
-	name = "SL L-204 laser gun"
-	desc = "A basic energy-based laser gun that fires concentrated beams of light which pass through glass and thin metal."
+	name = "E-20 mining emitter"
+	desc = "A basic mining laser that fires concentrated beams of light which break down rock. Notably, these beams of light melt down flesh as well."
```
```suggestion
	desc = "A basic mining tool that fires concentrated bolts of light, which easily cause flesh, stone, and metal to yield."
```

Erikafox commented on code/modules/projectiles/guns/energy/laser.dm:
```diff
@@ -20,20 +21,32 @@
 /obj/item/gun/energy/laser/practice
 	name = "practice laser gun"
-	desc = "A modified version of the L-204 laser gun, this one fires less concentrated energy bolts designed for target practice."
+	desc = "A modified version of the 104 laser gun, this one fires less concentrated energy bolts designed for target practice."
```
```suggestion
	desc = "A modified version of the L104 laser gun, this one fires less concentrated energy bolts designed for target practice."
```

Erikafox commented on code/modules/projectiles/guns/manufacturer/nanotrasen_sharplite/ballistics.dm:
```diff
@@ -39,11 +40,129 @@
 
+/obj/item/gun/ballistic/automatic/pistol/podium
+	name = "Advantage PH46 Podium"
+	desc ="A heavy pistol chambered in the high-velocity 4.6mm cartridge, designed to defeat common body armor. Despite the powerful cartridge, it is known to be surprisingly controllable, though not necessarily lightweight. Sold only to major corporate clients. This example was manufactured for Vigilitas Interstellar."
```
```suggestion
	desc ="A heavy pistol chambered in the high-velocity 4.6mm cartridge, designed to defeat common body armor. Despite the powerful cartridge, it is known to be surprisingly controllable, though not necessarily lightweight. Sold only to major corporate clients. "
```

The manufacturer stamp has the VI bit down

Erikafox commented on code/modules/projectiles/guns/manufacturer/nanotrasen_sharplite/lasers.dm:
```diff
@@ -0,0 +1,399 @@
+	mob_overlay_icon = 'icons/obj/guns/manufacturer/inteq/onmob.dmi'
+
+	icon_state = "x12_inteq"
+	item_state = "x12_inteq"
```
Put these in an inteq file pls

Erikafox commented on code/modules/projectiles/guns/manufacturer/nanotrasen_sharplite/lasers.dm:
```diff
@@ -0,0 +1,399 @@
+	ammo_x_offset = 1
+	shaded_charge = TRUE
+
+/obj/item/gun/energy/laser/retro
```
This is defined in a previous file.

Erikafox commented on code/modules/projectiles/guns/misc/beam_rifle.dm:
```diff
@@ -28,7 +28,7 @@
 	w_class = WEIGHT_CLASS_BULKY
 	ammo_type = list(/obj/item/ammo_casing/energy/beam_rifle/hitscan)
-	internal_magazine = FALSE //prevents you from giving it an OP cell - WS Edit //shut up dumb nerd
+	internal_magazine = FALSE //prevents you from giving it an OP cell - WS Edit
```
```suggestion
	internal_magazine = FALSE //prevents you from giving it an OP cell
```

FalloutFalcon replied:
```suggestion
	internal_magazine = FALSE
```

Apogee-dev commented:
I don't think it would hurt to put the inteq X12 in IRMG-only faction cargo.

rye-rice committed:
change i should have commited 3 days ago

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

rye-rice committed:
Apply suggestions from code review

Co-authored-by: Erika Fox <94164348+Erikafox@users.noreply.github.com>
Signed-off-by: rye-rice <58402542+rye-rice@users.noreply.github.com>

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

rye-rice committed:
Merge branch 'basic_nt_weaponry' of https://github.com/rye-rice/Shiptest into basic_nt_weaponry

generalthrax committed:
Thraxbranch (#13)

* min recoil and cadejo / barghest cases

* number changes + VI deepstorage / kiwi fixes

* mapstuff (NOT CAPTAIN WEAPONS)

* cargo

* supplypacks

* hos gun

* maps + no_cell

* bleh

* maps

* maps

rye-rice committed:
sprites

rye-rice committed:
Merge remote-tracking branch 'origin/basic_nt_weaponry' into basic_nt_weaponry

rye-rice committed:
inteq resolution + error fix

rye-rice committed:
oops

rye-rice committed:
yea

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

rye-rice committed:
repath 1

rye-rice committed:
repaths part 2

generalthrax committed:
teeny (#14)

generalthrax commented:
we put some work into this, should be compliant with reviews now

DIB-DOG commented on code/modules/projectiles/guns/manufacturer/nanotrasen_sharplite/ballistics.dm:
```diff
@@ -87,4 +207,234 @@ NO_MAG_GUN_HELPER(automatic/pistol/commander)
+	start_empty = TRUE
+
+/obj/item/ammo_box/magazine/wt550m9/ap
+	name = "wt550 magazine (4.6x30mm AP)"
```
Assuming this is meant to say Resolution vs WT-550? Pretty niche item, spawning loaded with AP, but it caught my eye.

generalthrax replied:
i gotcha

generalthrax committed:
Iconfixies (#15)

* fixies

* fixies

* there

* resolution

generalthrax committed:
Fuck (#16)

* first

* mauler

generalthrax committed:
Fixedd (#17)

* adds some crop tops (#5279)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request
<img width="498" height="406" alt="image"
src="https://github.com/user-attachments/assets/62684d1c-3938-4deb-8fd7-778b6f55fee6"
/>

## Why It's Good For The Game

more underwear options nice i thinkl

## Changelog

:cl:
add: three new croptops
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

* Automatic changelog generation for PR #5279 [ci skip]

* Automatic changelog compile [ci skip]

* lets headphones and earmuffs layer over hair when in the ear slot (#5262)

## About The Pull Request

<img width="67" height="80" alt="image"
src="https://github.com/user-attachments/assets/d80ef473-779c-4f14-bcc5-803a88d11234"
/>
<img width="77" height="109" alt="image"
src="https://github.com/user-attachments/assets/befdaac3-a205-4054-9376-429515a556f7"
/>


## Why It's Good For The Game

you cant put the headphone thingy under your hair thats not how it works

## Changelog

:cl:
fix: Earmuffs and headphones now layer over hair
/:cl:

* Automatic changelog generation for PR #5262 [ci skip]

* Chem Cleanup 2025 (#5162)

## About The Pull Request
Rips out like 3 chem systems of random healing chems. Adds new healing
chems

Indomide
 - Slow brute healing
 - Increases bleed rate
 - synergizes with eating food
 - copper, 2 part acetone, phosphorus
 
 Hadrakine
  - Instant brute healing
  - slow healing over time
  - overdose causes extremely reduced bleeding
  - 3 part alum, hydrogen, oxygen, sulferic acid

  
  Silfrine
  - High brute healing, scales off how hurt you are
  - Can seal low-tier wounds when applied via injection
  - slows you down on account of the pain and agony
- 3 part silver, phenol, carbon, sulpheric acid, 10u plasma catalyst and
450k heat
  
  
Alvitane
 - low burn healing
 - cools you off
 - can heal very low-level burn wounds
 -carbon, silicon, 2 part silver, cryptobiolin catalyst (5u), low temp
 
 Quardexane
 - moderate burn healing
 - instant burn heal when applied via injection
 - does some wound healing
 - 3 part cryostylane, bromine, lye
 
 Ysiltane 
 - High instant burn healing
 - Fuckton of stam damage that's right you gotta wait it out
 -ammonia, silver, sulfer, hydrogen, chlorine, 10u plasma catalyst
 
Charcoal
- it's charcoal, practically unchanged

Pancrazine 
- It's basically seiver
-Nitrogen, silicon, potassium, 5u ammonia base

Gjalrazine 
- Instant tox heals in exchange for disgust and other problems
-3 part pancrazine, 4 part carbon, 3 part diethylamine, 1u plasma
catalyst

Surviving oxygen chems
- Salbutamol
- Dexalin
Both are unchanged

Reflavored
-Omnizine
-Tric



## Why It's Good For The Game

We have way too many random chems that do the same thing. I wanna change
the focus away from station chem to our own thing.

## Changelog

:cl:
add: new chemicals, see pr 5162 on the git
del: remove hot potatoes
del: 3 eras of random chems, from libital to salicylic acid
/:cl:

---------

Signed-off-by: Erika Fox <94164348+Erikafox@users.noreply.github.com>
Co-authored-by: generalthrax <139387950+generalthrax@users.noreply.github.com>
Co-authored-by: Ical <86125936+Ical92@users.noreply.github.com>
Co-authored-by: ritorizo <41831966+ritorizo@users.noreply.github.com>
Co-authored-by: FalloutFalcon <86381784+FalloutFalcon@users.noreply.github.com>

* Automatic changelog generation for PR #5162 [ci skip]

* [Vigilitas Interstellar] Adds the Shrike-class Fast Response Frigate (#5087)

## About The Pull Request

Adds the Shrike-class Fast Response Frigate, a Vigilitas Interstellar
vessel centered around combat (Of the PvE and PvP variety), and security
contracting as a vessel of the largest security contractor in the
Galaxy, with 2 new VI outfits (Captain, and 2IC (Sergeant for this
vessel)). Weaponry currently present within are placeholders until the
Sharplite rework is merged.

***
>_Description is currently a WIP_
><img width="574" height="766" alt="image"
src="https://github.com/user-attachments/assets/c0ebf8b7-b9e5-41fc-91cf-317694ec547e"
/>
<details>
  <summary>Strongdmm photos</summary>
  
  <p align="center">Areas:</p>
<img width="1152" height="1120" alt="image"
src="https://github.com/user-attachments/assets/ffd09854-eccc-4f22-9695-572089bb933c"
/>

  <p align="center">Without Areas:</p>
<img width="1152" height="1120" alt="image"
src="https://github.com/user-attachments/assets/58e92054-a303-4dd5-921d-6cca1fa5fdab"
/>
</details>

  


## Why It's Good For The Game

Whilst Vigilitas may already have a combat related vessel ingame, it is
unfortunately locked to the Loss Prevention department of VI, instead of
a more broad generalist vessel. This addition changes that, giving VI
players who may have wanted to play the faction outside of Nanotrasen
and N + S Shipping vessel, without having to make a character who may
not potentially fit with the LP department.

## Changelog

:cl:
add: Adds the Shrike-class Fast Response Frigate, a new Vigilitas
Interstellar vessel.
add: Adds two additional VI outfits.
add: New secure safe subtype
/:cl:

---------

Signed-off-by: Burning02 <141873051+Burning02@users.noreply.github.com>
Co-authored-by: Erika Fox <94164348+Erikafox@users.noreply.github.com>

* Leyak Paperwork Buff + windoor (#5277)

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may
not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull
request process. -->

## About The Pull Request
I changed the trashcan in the bridge for a photocopier and made the
armoury windoor rotated inward and put a public one on the outside so
you can put stuff on the table like the HOP line

<img width="325" height="319" alt="image"
src="https://github.com/user-attachments/assets/ca775cee-8707-4ae7-a944-6496bf62492d"
/>

<img width="319" height="339" alt="image"
src="https://github.com/user-attachments/assets/6c1b65b8-c429-470e-a4f9-da8cf870eda3"
/>


## Why It's Good For The Game
photocopiers are like godsgrace to the land
<!-- Please add a short description of why you think these changes would
benefit the game. If you can't justify it in words, it might not be
worth adding. -->

## Changelog

:cl:
add: The Leyak has a photocopier now. It also has it's indoor adjusted.

/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put
your name to the right of the first :cl: if you want to overwrite your
GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the
icon ingame) and delete the unneeded ones. Despite some of the tags,
changelogs should generally represent how a player might be affected by
the changes rather than a summary of the PR's contents. -->

* holster + BDM

* Automatic changelog generation for PR #5277 [ci skip]

* Tiny Arrowsong Tweaks (#5275)

## About The Pull Request

- All-in-One tcomms in the command area
- Saluki and ammo in the ERT armoury
- Facemasks in both CLIP and Inteq ERT areas
- Defibrillator in ERT corpsman
- Renamed 7.5mm ammo can path back to 6.5 cause it was annoying
- Folders and Clipboards to the fax area
- Takes ammo out of the museum guns (keeps magazines)

## Why It's Good For The Game

Stuff to equip ERTs with

## Changelog

:cl:
add: Added some misc supplies and changes to Arrowsong's ERT armoury and
museum. Comms to Arrowsong
/:cl:

* Automatic changelog generation for PR #5275 [ci skip]

* shrike

---------

Signed-off-by: Erika Fox <94164348+Erikafox@users.noreply.github.com>
Signed-off-by: Burning02 <141873051+Burning02@users.noreply.github.com>
Co-authored-by: thearbiber <135081923+thearbiber@users.noreply.github.com>
Co-authored-by: Changelogs <action@github.com>
Co-authored-by: Erika Fox <94164348+Erikafox@users.noreply.github.com>
Co-authored-by: Ical <86125936+Ical92@users.noreply.github.com>
Co-authored-by: ritorizo <41831966+ritorizo@users.noreply.github.com>
Co-authored-by: FalloutFalcon <86381784+FalloutFalcon@users.noreply.github.com>
Co-authored-by: Burning02 <141873051+Burning02@users.noreply.github.com>
Co-authored-by: Ratvar <61395690+Ratvarr@users.noreply.github.com>

rye-rice committed:
Revert "Fixedd (#17)"

This reverts commit d65bb2da41d3728c669e75b86779788197e7fc27.

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

generalthrax committed:
Neuew (#18)

* shrike + bdm

* sniper spread

* x12

Aquidu commented:
results from my round of testing da guns:

harrier starts with an eohoma cell rather than a sharplite cell in the captain's locker

naming convention and gun silhouettes are very similar and a bit hard to tell what one is from a glance, i love the sprites but maybe a little bit more telling names or more distinct features could help? tbf it's also only the first round so maybe i just need time to get used to it. idk, just rambling

the power meter is very small and can make it hard to tell how much power is left

i think some of the back sprites might be broken? need to check again, don't remember which

advantage guns seem to need a little bit more before they're done? not a big deal but worth nothing i guess

things i loike:

the sprites are really cool and unique. very futuristic

i love the sounds and new bolt sprites, they sound way more appealing and satisfying to use

nt now has more intermediate options for weapons

anyway despite everything i said i'd say this pr is like, 95% good and awesome and things i like more than my criticisms, i am very glad this exists and thank you for making it.

generalthrax commented:
will fix mapping issues when i can

FalloutFalcon commented on code/game/objects/items/attachments/long_scope.dm:
```diff
@@ -9,21 +9,18 @@
 	gun.zoom_amt = initial(gun.zoom_amt)
 	gun.zoom_out_amt = initial(gun.zoom_out_amt)
-	gun.min_recoil_aimed = initial(gun.min_recoil_aimed)
 	gun.aimed_wield_slowdown = initial(gun.aimed_wield_slowdown)
```
this still needs to be reverted

thgvr replied:
yeah we won't be merging a removal of this.

generalthrax replied:
pretty sure i re-added this but i must've forgot

FalloutFalcon commented on code/game/objects/items/attachments/short_scope.dm:
```diff
@@ -9,21 +9,18 @@
 	. = ..()
 	gun.zoom_amt = initial(gun.zoom_amt)
 	gun.zoom_out_amt = initial(gun.zoom_out_amt)
-	gun.min_recoil_aimed = initial(gun.min_recoil_aimed)
```
ditto

FalloutFalcon commented on code/modules/projectiles/ammunition/energy/_energy.dm:
```diff
@@ -7,6 +7,5 @@
 	var/select_name = "energy"
 	resistance_flags = INDESTRUCTIBLE
 	fire_sound = 'sound/weapons/gun/laser/nt-fire.ogg'
-	firing_effect_type = /obj/effect/temp_visual/dir_setting/firing_effect/energy
```
?

generalthrax replied:
I may be mistaken but I believe this prevents doubling up on firing effect visuals

generalthrax replied:
yeah i was right. firing effect is handled elsewhere and it doubles up if this is put back in

generalthrax committed:
Shrikestuff (#19)

* harrier, shrike, and resolution attachment fix. expedition magazine fix

* names the hades

generalthrax committed:
minrecoilstuff (#20)

generalthrax commented:
should be everything addressed

Erikafox committed:
Merge remote-tracking branch 'upstream/master' into pr/5210

generalthrax committed:
Names (#21)

* x46 simplemob + passgrille

* names

* deepstorage

generalthrax committed:
Fixed (#22)

* akita + slot flags

* bulky Rush

* carg

* -

* x12onmob

* suitstore + x12

rye-rice committed:
fixes missing

rye-rice committed:
resprites

rye-rice committed:
fixed x01 sprites

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

rye-rice committed:
possibly last repath

rye-rice committed:
decreases spread to cm5's

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

FalloutFalcon commented on code/modules/projectiles/boxes_magazines/external/pistol.dm:
```diff
@@ -68,8 +68,8 @@
-	icon_state = "brazilpistol_mag-1"
-	base_icon_state = "brazilpistol_mag"
+	icon_state = "Zip_mag-1"
+	base_icon_state = "Zip_mag"
```
why is the icon state name capitalized...

generalthrax replied:
forgor

FalloutFalcon commented on code/modules/projectiles/guns/energy/energy_gun.dm:
```diff
@@ -1,11 +1,11 @@
-/obj/item/gun/energy/e_gun
+/obj/item/gun/energy/e_gun/
```
nit. i assume your trying to use this to represent that its an abstract class but its applied no where and ends up just looking like a mistake
```suggestion
/obj/item/gun/energy/e_gun
```

generalthrax replied:
whoopsie daisy

FalloutFalcon commented:
you should prob undraft this

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

rye-rice committed:
inteq podium

rye-rice committed:
x11 and polish

rye-rice committed:
conflicts

rye-rice committed:
mapfix

rye-rice committed:
inteq podium + podium in cargo yipeeee

rye-rice committed:
missing file

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry

generalthrax committed:
Descs + Podium + Misc Fixies (#23)

* Update TGS DMAPI

* Revert "Automatic TGS DMAPI Update"

* rhino

* rhino map

* locke

* pinscher gone + podium

* vanguard podium

* honorable vanguard pinscher

* descs

* shotgunpassflag

* zip icon fixiy

* w

* whoopsie daisy

---------

Co-authored-by: github-actions <action@github.com>

generalthrax committed:
Miningfacilitylasers (#24)

* Automatic changelog generation for PR #5311 [ci skip]

* bleeh

---------

Co-authored-by: Changelogs <action@github.com>

rye-rice commented:
needs final pass from apogee/spriter but should probably be fine in a RC state

Imaginos16 reviewed (approved):
Sorry it took me a bit, but from my analysis, this seems to be fine sprite-wise.

Apogee-dev reviewed (approved):
Everything here looks good.

rye-rice committed:
Merge remote-tracking branch 'upstream/master' into basic_nt_weaponry
